src/components/form-textarea/form-textarea.js

```diff
@@ -98,14 +102,19 @@ export default {
         (parseFloat(computedStyle.paddingBottom) || 0)
       // Calculate content height in "rows"
       const contentRows = (el.scrollHeight - offset) / lineHeight
-      // Put the old height back (needed when new height is equal to old height!)
-      el.style.height = oldHeight
```

Comment:
This line is needed to handle certain edge cases when changing the height.

Reply:
Note that this is altering the element directly, not returning a value.

Reply:
And this also triggers a reflow on the element.  Can't be placed inside an "if".